src/domains/ecommerce-common/03_storage.tf

```
@@ -73,50 +73,101 @@ module "ecommerce_storage" {
 
 resource "azurerm_storage_queue" "notifications_service_retry_queue" {
   name                 = "${local.project}-notifications-service-retry-queue"
-  storage_account_name = module.ecommerce_storage.name
-}
-
-resource "azurerm_storage_queue" "notifications_service_errors_queue" {
-  name                 = "${local.project}-notifications-service-errors-queue"
-  storage_account_name = module.ecommerce_storage.name
+  storage_account_name = module.ecommerce_storage_transient.name
 }
 
 resource "azurerm_storage_queue" "transactions_expiration_queue" {
   name                 = "${local.project}-transactions-expiration-queue"
-  storage_account_name = module.ecommerce_storage.name
+  storage_account_name = module.ecommerce_storage_transient.name
 }
 
 resource "azurerm_storage_queue" "transactions_close_payment_queue" {
   name                 = "${local.project}-transactions-close-payment-queue"
-  storage_account_name = module.ecommerce_storage.name
+  storage_account_name = module.ecommerce_storage_transient.name
 }
 
 resource "azurerm_storage_queue" "transactions_close_payment_retry_queue" {
   name                 = "${local.project}-transactions-close-payment-retry-queue"
-  storage_account_name = module.ecommerce_storage.name
+  storage_account_name = module.ecommerce_storage_transient.name
 }
 
 resource "azurerm_storage_queue" "transactions_refund_retry_queue" {
   name                 = "${local.project}-transactions-refund-retry-queue"
-  storage_account_name = module.ecommerce_storage.name
+  storage_account_name = module.ecommerce_storage_transient.name
 }
 
 resource "azurerm_storage_queue" "transactions_refund_queue" {
   name                 = "${local.project}-transactions-refund-queue"
-  storage_account_name = module.ecommerce_storage.name
+  storage_account_name = module.ecommerce_storage_transient.name
 }
 
 resource "azurerm_storage_queue" "transactions_notifications_retry_queue" {
   name                 = "${local.project}-transaction-notifications-retry-queue"
-  storage_account_name = module.ecommerce_storage.name
+  storage_account_name = module.ecommerce_storage_transient.name
 }
 
 resource "azurerm_storage_queue" "transactions_notifications_queue" {
   name                 = "${local.project}-transaction-notifications-queue"
-  storage_account_name = module.ecommerce_storage.name
+  storage_account_name = module.ecommerce_storage_transient.name
+}
+
+resource "azurerm_private_endpoint" "storage_deadletter_private_endpoint" {
+  count = var.env_short != "d" ? 1 : 0
+
+  name                = "${local.project}-storage-private-endpoint"
```

This should have a different name from the `transient` one
```suggestion
  name                = "${local.project}-deadletter-storage-private-endpoint"
```